get-resource.sh

```diff
@@ -10,10 +10,12 @@ export no_proxy="${no_proxy:-${NO_PROXY:-}}"
 
 # configurable variables
 SHARED_DIR="${SHARED_DIR:-/shared}"
-CURL_OUTPUT=""
-if [ "${CURL_VERBOSE:-}" = true ]; then
-    CURL_OUTPUT="--verbose"
-fi
+
+curl_with_flags() {
+    [ "${CURL_VERBOSE:-}" != 'true' ] || set -- --verbose "$@"
```

Comment:
nit: the double inverted logic caused me a headache

Reply:
Double inverted is a necessity here. Doing the same with `&&` causes `set -e` to trigger, while `||` catches it and executes the `set -- ...` part.

Reply:
```bash
[ "${CURL_VERBOSE:-}" = 'true' ] && set -- --verbose "$@" || :
```

may be more readable.

Yes, bash is ugly.

Reply:
"ternary" operation in Bash is not recommended as it has pitfalls. Here is might work, but better avoid the construct everywhere.

https://mywiki.wooledge.org/BashPitfalls number 22.

Reply:
could we do `if [ "${CURL_VERBOSE:-}" = 'true' ]; then set -- --verbose "$@"; fi` or are we trying to win at code golf? :slightly_smiling_face:

Reply:
@zaneb I pushed a more readable version, following your advice.

Back to you!